docs/getting-started/install-cent-node.md

```diff
@@ -45,13 +45,41 @@ For instructions to install `geth` on MacOS, see [Instructions for MacOS](https:
 ### System Requirements for Installing the Centrifuge Node:
 
 To accept the incoming P2P connections, you will need to open two ports for incoming TCP connections.
+ P2P Port: open ingress/egress
+ API Port: restrict at will, only you or your upstream systems should need to talk to it
 <!--
 **Resource Requirements for Centrifuge API Node**
 * 1 Gigabyte memory
 * 1 core
-* Allocate 100 Gigabyte of disk space (local copy of document data).
 -->
 
+### Running the Ethereum Rinkeby Node
+
+Rinkeby is a testnet
+
+**Resource Requirements**
+
+* 4 Gigabyte memory
+  * Syncing mode uses around 3 Gigabyte
+  * Normal sync uses around 400 Megabyte
+* 2 cores
+  * Syncing mode uses about 2-3 cores
+  * Normal mode uses about 0.2 cores
+* Allocated disk space of at least 100 Gigabyte
+  * Sync mode uses about 1 Gigabyte
+  * Full node requires about 50 Gigabyte
+* Syncing will take about 1-2 hours. This can vary depending on the resource availability of the full node(s).
+  * Optionally, you can use SSDs speed it up.
+
+Start the `geth` process, we chose running `go-ethereum` in light mode, for speed and simplicity, feel free to tailor this setup to fit your needs: 
```

Comment:
Start the `geth` process:

  ```bash
  $ geth --rinkeby --light --rpc --rpcapi db,eth,net,web3,txpool --ws \
  --wsorigins "*" --wsapi db,eth,net,web3,txpool > /tmp/geth.log 2>&1 &
   ```
This command runs `go-ethereum` in the _light_ mode for speed and simplicity. You can modify this command based on your requirements.